service/src/test/java/dk/dbc/search/work/presentation/service/BeanFactory.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2020 DBC A/S (http://dbc.dk/)
+ *
+ * This is part of work-presentation-service
+ *
+ * work-presentation-service is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * work-presentation-service is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package dk.dbc.search.work.presentation.service;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import javax.persistence.EntityManager;
+import javax.sql.DataSource;
+import javax.ws.rs.client.ClientBuilder;
+import org.glassfish.jersey.client.JerseyClientBuilder;
+
+/**
+ *
+ * @author Morten Bøgeskov (mb@dbc.dk)
+ */
+public class BeanFactory {
```

Comment:
Det er et ret generisk navn - kunne der evt. være dokumentation af hvad formålet med klassen er?

Reply:
Det er en bean factory til integration test. Jeg kan godt skrive dét. Det har bare indtryk af at der står noget for at der skal stå noget.

Reply:
Your call. Vi har jo ret forskellige tilgange til det, men jeg kan ikke argumentere hårdt for at min tilgang er bedre end din. Den hjælper mig, men jeg er vist et mindretal :-)